src/main/kotlin/com/atlassian/performance/tools/jiraactions/page/IssueCreateDialog.kt

```diff
@@ -49,12 +54,32 @@ internal class IssueCreateDialog(
     }
 
     fun fillRequiredFields(): IssueCreateDialog {
+
+        driver.wait(elementToBeClickable(configColumnField)).click()
+        val locator = By.xpath("//div[@class='qf-picker-header']/dl/dd/a[text()='All']")
```

Comment:
So this shows all issue fields in the dialog.
AFAIK the HWR XL7 dataset is problematic, because some projects/issue types have a mandatory `Resolution` field, which has no default value *and* it's not visible on the screens by default.

Reply:
Agree, the root cause is the dataset. Making all field available might add values for people using their own dataset.
Do we think we might need this in the future? If so I will create a new branch to pick those changes and raise a new PR.

Reply:
Let's raise a new PR. Let's include the context, the purpose of this change, because beforehand it wasn't clear. Ideally we'd write an integration test with such a dataset (funky issue screen, not necessarily XL), but if we need to cut scope, then at least let's explain it in the commit message or method Javadoc.

Reply:
sure, will do